security/maltrail/src/opnsense/scripts/OPNsense/Maltrail/setup.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+mkdir -p /var/log/maltrail/
```

Comment:
is it correct to assume maltrail is all log files, no other database / temporary storage location that might need an MFS fix?

Reply:
Indeed, nice catch!